keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -994,10 +994,16 @@ The conflict will be resolved by assigning precedence based on each
 A derived service will be accessible with the clusterset IP at the ports
 dictated by child services. If the external properties of service ports for a
 set of exported services don’t match, the clusterset service will expose the
-union of service ports declared on its constituent services. Should a port name
-be used for multiple non-identical (`port`, `protocol`, `appProtocol`) service
-ports by different constituent services, the conflict resolution policy will
-determine which values are used by the derived service.
+union of service ports declared on its constituent services.
```

Comment:
As a followup to this, it seemed like there was some debate over whether `union` should actually be `intersection` instead? Refs https://github.com/kubernetes-sigs/mcs-api/pull/78#issuecomment-2383809649

As currently specified, either some intelligent endpoint selection would need to be applied based on target port if some ports are not available on some endpoints (I'm not sure if this is possible/practical), or some requests would fail non-deterministically if routed to an endpoint lacking a given service port.

Reply:
Not entirely sure about the pros/cons/context of this decision, guessing we can discuss it at the next meeting